File: 3718-smallest-missing-multiple-of-k/3718-smallest-missing-multiple-of-k.cpp
class Solution {
public:
    int missingMultiple(vector<int>& nums, int k) {
    sort(nums.begin(),nums.end());
    int e;
    for(int i=0;i<nums.size();i++){
        if(find(nums.begin(),nums.end(),(i+1)*k)==nums.end()){
            return (i+1)*k;
        }
        if(nums[i]%k==0){
            e=nums[i];
        }
    }
    return e+k;

    }
};

File: 3718-smallest-missing-multiple-of-k/3718-smallest-missing-multiple-of-k.py
class Solution:
    def missingMultiple(self, nums: List[int], k: int) -> int:
        e=0
        for i in range(len(nums)):
            if((i+1)*k not in nums):
                return (i+1)*k
            if(nums[i]%k==0):
                e=nums[i]
        return e+k